core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.directive.ts

```diff
@@ -142,7 +138,10 @@ angular.module('oppia').directive('topNavigationBar', [
             return SidebarStatusService.isSidebarShown();
           };
           ctrl.toggleSidebar = function() {
-            SidebarStatusService.toggleSidebar();
+            // SidebarStatusService.toggleSidebar();
+            // $scope.update = !$scope.update;
+            $scope.toggle();
```

Comment:
why this is changed?
Due to change detection issues with upgradeComponent.
When toggleSidebar is called directly from there, change detection only occurs in ajs not in angular.
So, I changed this in such a way that both ajs and angular change detection work.